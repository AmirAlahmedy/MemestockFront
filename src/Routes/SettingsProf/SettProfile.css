h2{
    color: rgb(28, 28, 28);
    font-size: 20px;
    font-weight: 500;
    line-height: 24px;
    padding: 40px 0px;
    
}

.ayhaga-ay{
    max-width: 1200px;
    display: flex;
    /* margin: 0px auto; */
    /* padding: 0px 16px; */
    float:left;
    /* margin: 0 50px; */
    /* padding: 0px; */
    border: 0;
    font-size: 100%;
    font: inherit;
    vertical-align: baseline;
   
    
}
.profile-info{
    font-size: 15px;
    font-weight: 700;
    letter-spacing: 0.5px;
    line-height: 12px;
    text-transform: uppercase;
    color: rgb(124, 124, 124);
    margin-bottom: 32px;
    padding-bottom: 6px;
    border-bottom: 2px solid rgb(237, 239, 241);
}
.display-box{
    margin-top: 12px;
    flex-direction: column;
    align-items: flex-start;
}
.display-info{
    font-size: 20px;
    font-weight: 500;
    line-height: 20px;
    margin-bottom: 4px;
    color: rgb(124, 124, 124);
}
.display-box p{
    font-size: 12px;
    font-weight: 300;
    line-height: 16px;
    color: rgb(124, 124, 124);
}
.ayhaga{
    max-width: 688px;
    flex-grow: 1;
    flex-shrink: 1;
    flex-basis: auto;
   

}
.optional-name{
    background-color: rgb(255, 255, 255);
    color: rgb(28, 28, 28);
    box-sizing: border-box;
    height: 48px;
    margin-bottom: 8px;
    width: 100%;
    border-width: 1px;
    border-style: solid;
    border-color: rgb(237, 239, 241);
    border-image: initial;
    border-radius: 4px;
    padding: 12px 24px 4px 12px;
}
.box{
    margin-top: 12px;
    flex-direction: column;
    align-items: flex-start;
}

.about-box{
    display: flex;
    flex-direction: column;
    margin-bottom: 32px;
}

.ab1{
    display: flex;
    flex-direction: column;
    max-width: 80%;  
}
.ab1 h3{
    font-size: 20px;
    font-weight: 500;
    line-height: 20px;
    margin-bottom: 4px;
    color: rgb(124, 124, 124);

}
.ab1 p{
    font-size: 12px;
    font-weight: 300;
    line-height: 16px;
    text-align: center;
    color: rgb(124, 124, 124);
    white-space: nowrap;
}
.ab2{
    -webkit-box-flex: 1;
    flex-grow: 1;
    display: flex;
    -webkit-box-pack: end;
    justify-content: flex-end;
    -webkit-box-align: center;
    align-items: center;
}
.ab2 textarea{
    /* background-color: rgb(255, 255, 255);
    box-sizing: border-box;
    color: rgb(28, 28, 28);
    display: block;
    width: 100%;
    border-width: 1px;
    border-style: solid;
    border-color: rgb(237, 239, 241);
    border-image: initial;
    border-radius: 4px;
    padding: 8px; */
    margin: 0 auto;
    border-radius:4px;
    border: 1px solid #ccc;
    resize: none;
    overflow-y: auto;
    overflow-x: hidden;
    font-size: 0.85em;
    padding: 1em;
    width: 100%;
}
.char{
    font-size: 12px;
    font-weight: 400;
    line-height: 16px;
    color: rgb(135, 138, 140);
}