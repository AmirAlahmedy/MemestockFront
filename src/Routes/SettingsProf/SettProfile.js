import React from 'react'
import './SettProfile.css'

const SettProfile = () => {
  return (
  //  <div class="ayhaga-ay">
    <div class="ayhaga">
             <h2>Customize Profile</h2>
              <h3 class="profile-info">Profile Information</h3>

              <div className="display-box">
                 <h3 class="display-info">Display Information</h3>


                  <p>Set a display name. This does not change your username.</p>


                  <div className="box">
                  <input class="optional-name" placeholder="Display Name (optional)" type="text"  maxLength="15" />
                  </div>

              </div>

              <div className="about-box">

                <div className="ab1">
                  <h3>About</h3>
                  <p>A brief description of yourself shown on your profile.</p>
                </div>

                <div className="ab2">
                  <textarea placeholder="About (optional)" maxLength="200" rows="5"></textarea>
                  </div>
                  <div class="char">200 Characters</div>
          

              </div>


        </div>
  //  </div>
  )
}

export default SettProfile
