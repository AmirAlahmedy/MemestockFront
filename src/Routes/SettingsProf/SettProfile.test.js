import React from "react";
import { create } from "react-test-renderer";
import SettProfile from "./SettProfile";

describe("SettProfile component snapshot", () => {
    test("it matches the snapshot", () => {
      const component = create(<SettProfile />);
      expect(component.toJSON()).toMatchSnapshot();
    });
  });

  it("Shows pass page meta", async () => {
    const component = create(<ChangePass />);
    const instance = component.getInstance();
    await instance.handleSubmit();
    console.log(instance.state); 
  });