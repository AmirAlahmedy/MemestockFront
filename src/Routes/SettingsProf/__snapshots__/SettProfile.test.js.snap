// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettProfile component snapshot it matches the snapshot 1`] = `
<form
  className="form-h"
  onSubmit={[Function]}
>
  <div
    class="ayhaga"
  >
    <h3
      class="profile-info"
    >
      Profile Information
    </h3>
    <div
      className="display-box"
    >
      <p>
        Set a display name. This does not change your username.
      </p>
      <input
        className="optionalName"
        id="displayname"
        placeholder="Display Name (optional)"
        type="text"
      />
    </div>
    <div
      className="about-box"
    >
      <div
        className="ab1"
      >
        <h3
          class="profile-info"
        >
          About
        </h3>
        <p>
          A brief description of yourself shown on your profile.
        </p>
      </div>
      <div
        className="ab2"
      >
        <textarea
          className="txtArea"
          id="about"
          onChange={[Function]}
          placeholder="About (optional)"
        />
      </div>
      <div
        class="char"
      >
        200 Characters
      </div>
      <input
        class="btn btn-primary"
        id="submit"
        type="submit"
        value="Save"
      />
    </div>
  </div>
</form>
`;
